cmd/clusterctl/cmd/validate_cluster.go

```diff
@@ -76,7 +77,9 @@ func RunValidateCluster() error {
 	if err = validation.ValidateClusterAPIObjects(os.Stdout, c, vco.KubeconfigOverrides.Context.Cluster, vco.KubeconfigOverrides.Context.Namespace); err != nil {
 		return err
 	}
+	if err = validation.ValidateKubeClusterObjects(os.Stdout, c, vco.KubeconfigOverrides.Context.Cluster); err != nil {
```

Comment:
The inline assignment should be `err :=` to avoid overwriting the `err` outside of the `if` scope.

Reply:
Noticed the aline above shows the same pattern, good opportunity to fix it there too :)